docs/site-example/site.conf

```diff
@@ -183,4 +183,9 @@
   -- setup_mode = {
   --  skip = true,
   -- },
+
+  -- Show/hide the Altitude Field
```

Comment:
One more minor note: please remove the capitalization of the words "altitude" and "field".